README.md

```diff
@@ -15,6 +15,16 @@ This is a [Node.js](https://nodejs.org/en/) module available through the
 $ npm install express-session
 ```
 
+## Debugging
+
+```sh
+$ export DEBUG=express-session
+```
+
+Setting this property prior to the run (use `set` in Windows)
+enables `debug` mode, and prints minimal but vital debug
```

Comment:
```suggestion
enables [`debug`](https://www.npmjs.com/package/debug) mode, and prints minimal but vital debug
```

I think linking to the debug package's docs is worth while here

Reply:
thanks, fixed it.